code/go/0chain.net/smartcontract/stakepool/edb_delegate_pool.go

```diff
@@ -21,7 +21,7 @@ func newDelegatePoolUpdate(poolID, pId string, pType spenum.Provider) *DelegateP
 	return &spu
 }
 
-func (dp DelegatePool) emitNew(
+func (dp DelegatePool) EmitNew(
```

Comment:
Seems unnecessary exporting, this method is only used in the package

Reply:
Using it in initial staking